### ISSUE TITLE
TOPPVIew on windows Chromatogram chrom.mzML file crashes

### ISSUE THREAD

**Issue by KyowonJeong** [OpenMS/OpenMS#6668]
Chromatogram file openning crashs when selected to visualization for the example file:

File -> Open example file -> Chromatograms -> Syogenes.chrom.mzML(and for others files the same crash happens)
In the bottom windows, select "views" and if any item (e.g., Peptide 0 559.788 AAGGISSLEDAK 0" is selected, it crashes.
Screenshot: 

<img width="1489" alt="Screenshot 2023-02-05 at 6 32 28 PM" src="https://user-images.githubusercontent.com/38280645/216939446-22d692dd-a4c4-4a5f-8ae0-4b47341a50f2.png">

**Comment by timosachsenberg** [comment]
can confirm:
```
#0  __cxxabiv1::__cxa_throw (obj=0x7fffe4011a60, tinfo=0x7ffff7f7f110 <typeinfo for OpenMS::Exception::InvalidRange>,
    dest=0x7ffff7a5f6d0 <OpenMS::Exception::InvalidRange::~InvalidRange()>)
    at /opt/conda/conda-bld/gcc-compiler_1654084175708/work/gcc/libstdc++-v3/libsupc++/eh_throw.cc:77
#1  0x00007ffff799d0f7 in OpenMS::DimMZ::map(OpenMS::ChromatogramPeak const&) const ()
   from /home/sachsenb/OMS/OpenMS/openms-build/lib/libOpenMS_GUI.so
#2  0x00007ffff7b758c7 in OpenMS::Painter1DChrom::paint(QPainter*, OpenMS::Plot1DCanvas*, int) ()
   from /home/sachsenb/OMS/OpenMS/openms-build/lib/libOpenMS_GUI.so
#3  0x00007ffff7b96284 in OpenMS::Plot1DCanvas::paint(QPainter*, QPaintEvent*) ()
   from /home/sachsenb/OMS/OpenMS/openms-build/lib/libOpenMS_GUI.so
#4  0x00007ffff7b968f0 in OpenMS::Plot1DCanvas::paintEvent(QPaintEvent*) ()
   from /home/sachsenb/OMS/OpenMS/openms-build/lib/libOpenMS_GUI.so
#5  0x00007ffff535b795 in QWidget::event(QEvent*) () from /home/sachsenb/miniconda3/lib/libQt5Widgets.so.5
#6  0x00007ffff5330ca0 in QApplicationPrivate::notify_helper(QObject*, QEvent*) ()
   from /home/sachsenb/miniconda3/lib/libQt5Widgets.so.5
#7  0x00007ffff5336b19 in QApplication::notify(QObject*, QEvent*) () from /home/sachsenb/miniconda3/lib/libQt5Widgets.so.5
#8  0x00007ffff7aa83db in OpenMS::QApplicationTOPP::notify(QObject*, QEvent*) ()
   from /home/sachsenb/OMS/OpenMS/openms-build/lib/libOpenMS_GUI.so
#9  0x00007ffff4b8e28a in QCoreApplication::notifyInternal2(QObject*, QEvent*) ()
   from /home/sachsenb/miniconda3/lib/libQt5Core.so.5
#10 0x00007ffff5356445 in QWidgetPrivate::sendPaintEvent(QRegion const&) ()
   from /home/sachsenb/miniconda3/lib/libQt5Widgets.so.5
```

**Comment by timosachsenberg** [comment]
@cbielow this might be related to the mapping code

**Comment by cbielow** [comment]
> @cbielow this might be related to the mapping code

Indirectly, yes. The TVController tries to open a chromatogram layer as a 1D spectrum... that's not going to end well.
Fixed in #6673 among a few other things.

**Comment by KyowonJeong** [comment]
On windows 10, it works nice. And the viz looks very nice as well.
